x-pack/plugins/ml/public/util/chart_utils.js

```diff
@@ -228,6 +228,15 @@ export function getTickValues(startTimeMs, tickInterval, earliest, latest) {
   return tickValues;
 }
 
+const LABEL_WRAP_THRESHOLD = 60;
+
+// Checks if the string length of a chart label (detector description
+// and entity fields) is above LABEL_WRAP_THRESHOLD.
+export function isLabelLengthAboveThreshold({ detectorLabel, entityFields }) {
```

Comment:
this function is using character counting to measure whether it should wrap. i don’t think the font is monospaced, which will result in inconsistent results.